satellite/gracefulexit/endpoint.go

```diff
@@ -281,20 +301,16 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 
 				exitStatusRequest.ExitSuccess = false
 				// TODO needs signature
-				transferMsg = &pb.SatelliteMessage{
-					Message: &pb.SatelliteMessage_ExitFailed{
-						ExitFailed: &pb.ExitFailed{
-							Reason: pb.ExitFailed_OVERALL_FAILURE_PERCENTAGE_EXCEEDED,
-						},
-					},
+				transferMsg, err = endpoint.getFinishedMessage(ctx, endpoint.signer, nodeID, exitStatusRequest.ExitFinishedAt, exitStatusRequest.ExitSuccess, pb.ExitFailed_OVERALL_FAILURE_PERCENTAGE_EXCEEDED)
+				if err != nil {
+					return rpcstatus.Error(rpcstatus.Internal, err.Error())
 				}
 			} else {
 				exitStatusRequest.ExitSuccess = true
 				// TODO needs signature
```

Comment:
the `getFinishedMessage` call should add the signature right, so I think this TODO can be taken out?

Reply:
Oops. TODO removed